model/src/index.ts

```diff
@@ -7,8 +7,10 @@ export * from '~/src/conditions/index.js'
 export * from '~/src/pages/index.js'
 export * from '~/src/utils/helpers.js'
 export * from '~/src/utils/markdown.js'
+export * from '~/src/common/pagination/index.js'
```

Comment:
Super non-blocker, but these exports are currently ordered A-Z

Maybe there's an ESLint option we could add (similar to [`import/order`](https://github.com/DEFRA/forms-designer/blob/main/.eslintrc.cjs#L95-L96))

Reply:
Really didn't spot that haha. I think this would make a nice-to-have tech debt ticket...